spt/OrangeBox/overlay/overlays.cpp

```diff
@@ -24,6 +24,28 @@ CameraInformation rearViewMirrorOverlay()
 	return info;
 }
 
+CameraInformation havokViewMirrorOverlay()
+{
+	CameraInformation info;
+	Vector add(0, 0, 64);
+	Vector* pos = &(vphysicsDLL.PlayerHavokPos + add);
```

Comment:
You're taking an address of a temporary value that is immediately dropped, leading to UB